wire-swift-generator/api/wire-swift-generator.api

```diff
@@ -4,13 +4,21 @@ public final class com/squareup/wire/swift/SwiftGenerator {
 	public final fun generateTypeTo (Lcom/squareup/wire/schema/Type;Lio/outfoxx/swiftpoet/FileSpec$Builder;)V
 	public final fun generatedTypeName (Lcom/squareup/wire/schema/Type;)Lio/outfoxx/swiftpoet/DeclaredTypeName;
 	public static final fun get (Lcom/squareup/wire/schema/Schema;Ljava/util/Map;)Lcom/squareup/wire/swift/SwiftGenerator;
+	public final fun getSafeDeclaredTypeName (Lcom/squareup/wire/schema/Type;)Lio/outfoxx/swiftpoet/DeclaredTypeName;
 	public final fun getSchema ()Lcom/squareup/wire/schema/Schema;
 }
 
 public final class com/squareup/wire/swift/SwiftGenerator$Companion {
 	public final fun builtInType (Lcom/squareup/wire/schema/ProtoType;)Z
 	public final fun get (Lcom/squareup/wire/schema/Schema;Ljava/util/Map;)Lcom/squareup/wire/swift/SwiftGenerator;
 	public static synthetic fun get$default (Lcom/squareup/wire/swift/SwiftGenerator$Companion;Lcom/squareup/wire/schema/Schema;Ljava/util/Map;ILjava/lang/Object;)Lcom/squareup/wire/swift/SwiftGenerator;
+	public final fun getBUILT_IN_TYPES ()Ljava/util/Map;
```

Comment:
I spoke to @oldergod offline about this to lead up to this PR but yes will wait for the approval here.

Reply:
@oldergod can we specify the name to expose for this tool so its not so ugly `getBUILT_IN_TYPES`?

Reply:
You can add `@JvmStatic` to the variable.

But instead of exposing the list of built-in types, could we just expose a method like `fun builtInType(protoType: ProtoType): Boolean = protoType in BUILT_IN_TYPES.keys` ?

Reply:
ha that is already exposed, ok will update this `fun builtInType(protoType: ProtoType): Boolean = protoType in BUILT_IN_TYPES.keys`